build/unix.rs

```diff
@@ -8,6 +8,7 @@ fn stream_cc_platform_specifics(build: &mut cc::Build, wwise_sdk: &PathBuf) -> i
         .compiler("clang")
         .shared_flag(true)
         .opt_level(2)
+        .flag("-std=c++17")
```

Comment:
Could you explain why you added this? Was it a mistake on my part you went on and fixed, or is it something you shouldn't have committed? Rrise on Linux was already working without this flag, so I'm unsure this is required?

Reply:
Not including this flag gives errors that C++11 Extension is required. But since Linux works, I just removed the `!include('unix.rxs')` and included the function `stream_cc_platform_specifics` as it's own in macos.rs

Reply:
Aaah I see. Then I think what you did was actually right; I prefer limiting code duplication if it can be avoided. According to the [wwise docs](https://www.audiokinetic.com/en/library/edge/?source=SDK&id=projectsettings.html), only C++11 is required for Wwise 2021, and C++14 with Wwise 2022 (on Mac, build requirements are Xcode 14.0 with macOS SDK 12.3, for v10.9 Mavericks deployment target at minima).
CLang 3.4+ uses C++14 by default, but you could try setting it manually to `std=c++14` and see if it still works. What feature of C++17 was needed in order to build? I'm inclined to add the flag requirement to unix.rs (or at least c++11 for the Wwise 2021 branch, and c++14 for the Wwise 2022 branch, as the Wwise docs say) if it works for you too.

If it doesn't work, then I'd suggest you just add the flag in unix.rs, conditionally compiled if the target is macos. This is a bit convoluted, but would remove code duplication... what do you think?

Reply:
I see. Yeah it works with c++14, so I've added it in unix.rs and included it again in macos.rs 😄